docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: "3.7"
+
+services:
+  pgdb:
+    build: env/postgres
+    image: pgdb
+    ports:
+      - "5432:5432"
+    environment:
+      - POSTGRES_PASSWORD=pw
+
+  client:
+    depends_on:
+      - pgdb
+    build: env/client
+    image: client
+    volumes:
+      - $PWD/tmp:/mount
```

Comment:
Here is where the mounting is setup. In my case, I created a `tmp` directory at the top level of the repo folder, so this will mount the `tmp` directory inside the container at `/mount`. Anything inside there will show up in the container.

Reply:
So to clarify, my file structure is `csci-540-fall2021/tmp`, and inside the `tmp` dir I have `run_sql`, `up_psql`, and `lecture.sql`. I didn't include the SQL file in the PR since it's my own, but after pulling this branch, your local structure should match that. And when you run  `docker-compose run client`, you'll be able to `cd /mount`, and run those commands a-okay.